sdk/core/core-client-rest/test/urlHelpers.spec.ts

```diff
@@ -107,7 +107,7 @@ describe("urlHelpers", () => {
 
     assert.equal(
       result,
-      `https://example.org/foo?existing=hey&arrayQuery=ArrayQuery1%2Cbegin%21*%27%28%29%3B%3A%40+%26%3D%2B%24%2C%2F%3F%23%5B%5Dend%2C%2C`,
+      `https://example.org/foo?existing=hey&arrayQuery=ArrayQuery1,begin%21*%27%28%29%3B%3A%40+%26%3D%2B%24%2C%2F%3F%23%5B%5Dend,,`,
```

Comment:
Note this change: previously the array separator comma was being URL-encoded, but with this PR it now isn't. This feels like a bugfix to me, but not 100% sure?